js/repeater.js

```diff
@@ -603,7 +609,13 @@
 								if(index<renderer.nested.length){
 									loopNested(cont, index);
 								}else{
-									proceed('complete', args);
+									if (++skip % 50 === 0) {
```

Comment:
The problem is that we have an API that is based on callbacks for the render / sub-render steps. I tried a lot of different solutions but kept running into the same exponential growth because of that. Prior to realizing I could use setTimeout as a sort of trick/hack I was feeling like we were going to have to deprecate the API and start over (which might still be worthwhile but is obviously a breaking change)